homeassistant/components/tuya/light.py

```diff
@@ -210,14 +220,10 @@ class TuyaLightEntityDescription(LightEntityDescription):
     "sp": (
         TuyaLightEntityDescription(
             key=DPCode.FLOODLIGHT_SWITCH,
+            icon="mdi:light-flood-down",
```

Comment:
This is a primary entity, we generally do not provide icons for those. Additionally, these chips are reused for other things and thus may not be accurate in this specific case.

Let's not add an icon here.